setup.py

```diff
@@ -76,6 +76,7 @@
         "filelock",
         "stopit",
         "tabulate",
+        "py-tes",
```

Comment:
I don't think this needs to be a required dependency. But the docs should probably say that py-tes needs to be installed.

Reply:
And the import statement [here](https://github.com/snakemake/snakemake/pull/1127/files#diff-a1865e2681ce5d0b775d62c336fe1ac260c6855d91715e5bb033a2babaae830dR40) should be wrapped in a try-except with a `WorkflowError("Unable to import Python package tes. TES backend requires py-tes to be installed. Please install py-tes, e.g. via Conda or Pip.")`

Reply:
Ok, I added a try-except statement in the __init__ function. I will also a note at the documentation